extensions/skyportal/skyportal/handlers/api/archive.py

```diff
@@ -0,0 +1,163 @@
+from penquins import Kowalski
+
+from baselayer.log import make_log
+from baselayer.app.access import auth_or_token
+from baselayer.app.env import load_env
+from ..base import BaseHandler
+
+
+env, cfg = load_env()
+log = make_log("archive")
+
+
+gloria = Kowalski(
+    token=cfg["app.gloria.token"],
+    protocol=cfg["app.gloria.protocol"],
+    host=cfg["app.gloria.host"],
+    port=int(cfg["app.gloria.port"]),
+    timeout=10,
+)
+log(f"Gloria connection OK: {gloria.ping()}")
+
+
+class ArchiveCatalogsHandler(BaseHandler):
+    @auth_or_token
+    def get(self):
+        query = {"query_type": "info", "query": {"command": "catalog_names"}}
```

Comment:
This query will need to change to filter for only the light curve catalogs if we decide to implement dmitryduev/kowalski#60, right? Maybe just leave a TODO comment here to that effect

Reply:
No, this should return everything available, in case we decide to expose more of K's functionality through Fritz. The specific handlers below (like `ArchiveHandler.GET`) filter out the unnecessary stuff.